src/index.html

```diff
@@ -5,9 +5,51 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="i-mac"
+        class="card__image"
+      >
+      <h3 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__details">
+        <div class="stars">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
+
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
```

Comment:
`<div class="stars">` is redundant
```suggestion
      <div class="card__details">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>

        <p class="card__reviews">Reviews: 5</p>
      </div>
```